src/base/GasUseAnalysis.ml

```diff
@@ -773,7 +775,7 @@ struct
 
     let tvar a = TypeVar a in
     (* Make a simple identifier of type 'A *)
-    let si a = ER.mk_id (mk_ident a) (tvar "'A") in
+    let si a = ER.mk_id (asId a) (tvar "'A") in
```

Comment:
Seems like we only use `mk_id` with `asId` of the first parameter. Perhaps this deserves to be the main version of this function: `ER.mk_id a (tvar "'A")`? Feel free to ignore this, not a big deal anyways.